app/AcquireIntersonArrayBMode.cxx

```diff
@@ -147,13 +147,18 @@ int main( int argc, char * argv[] )
     std::cerr << "Invalid requested depth for probe." << std::endl;
     }
 
+  const double ns = ContainerType::MAX_SAMPLES; // number of samples
+  const double fs = frequencies[ frequencyIndex ] / 1000; // [kHz]=[samples/ms] - frequency per sample
+  double depth = c * ns / ( 2 * fs );
+
   const int scanWidth = container.GetWidthScan();
   const int scanHeight = container.GetHeightScan();
   std::cout << "Width = " << width << std::endl;
   std::cout << "Height = " << height << std::endl;
   std::cout << "ScanWidth = " << scanWidth << std::endl;
   std::cout << "ScanHeight = " << scanHeight << std::endl;
   std::cout << "MM per Pixel = " << container.GetMmPerPixel() << std::endl;
```

Comment:
I think these should be removed, because they are scan converter related, which is not relevant to our output.